docs/workspaces.md

```diff
@@ -244,20 +248,28 @@ The `subPath` specified in a `Pipeline` will be appended to any `subPath` specif
 
 Sharing a `Workspace` between `Tasks` requires you to define the order in which those `Tasks`
 will be accessing that `Workspace` since different classes of storage have different limits
-for concurrent reads and writes. For example, a `PersistentVolumeClaim` might only allow a
-single `Task` writing to it at once.
-
-**Warning:** You *must* ensure that this order is correct. Incorrectly ordering can result
-in a deadlock where multiple `Task` `Pods` are attempting to mount a `PersistentVolumeClaim`
-for writing at the same time, which would cause the `Tasks` to time out.
-
-To define this order, use the `runAfter` field in your `Pipeline` definition. For more
+for concurrent reads and writes. For example, a `PersistentVolumeClaim` with
+[access mode](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes)
+`ReadWriteOnce` only allow `Tasks` on a the same node writing to it at once.
+
+Using parallel `Tasks` in a `Pipeline` will work with `PersistentVolumeClaims` configured with
+[access mode](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes)
+`ReadWriteMany` or `ReadOnlyMany` but you must ensure that those are available for your storage class.
+When using `PersistentVolumeClaims` with access mode `ReadWriteOnce` for parallel `Tasks`, you can configure a
+workspace with it's own `PersistentVolumeClaim` for each parallel `Task`. See a full example of a 
+[Pipeline with parallel tasks using PersistentVolumeClaims](../examples/v1beta1/pipelineruns/pipelinerun-with-parallel-tasks-using-pvc.yaml).
+
+Use the `runAfter` field in your `Pipeline` definition to define when a `Task` should be executed. For more
 information, see the [`runAfter` documentation](pipelines.md#runAfter).
 
+**Warning:** You *must* ensure that this order is compatible with your configured access modes for your `PersistentVolumeClaim`.
+Parallel `Tasks` using the same `PersistentVolumeClaim` with access mode `ReadWriteOnce`, may execute on
+different nodes and be forced to execute sequentially which may cause `Tasks` to time out.
```

Comment:
Does the k8s scheduler force this sequential ordering automatically or is it something the _user_ is forced to do by adding `runAfter`?

Reply:
The k8s scheduler puts pods on nodes - immediately, they all mount the PVC. But the mount only succeeds on **one node** at the time. Pods on other nodes is stuck in state "Pending" - but they eventually start to run when the volume is not used on other nodes anymore.

And when the pods in the "other nodes" gets into "Running" - the Task is already started - so it may have "timed out" as was already documented.

But on my single node cluster - all tasks can run concurrently, so I did not see that parallel tasks on other nodes may not start until the volume is available.

I did not know how this was working, so I had to create a bigger test to validate this. Me and @skaegi discussed about it in the last API WG.